### ISSUE TITLE
UX: Click to copy may be unintuitive

### ISSUE THREAD

**Issue by aahern** [AlexPiccana/color-generator#7]
Click to copy gives no user feedback currently, and may result in the user unintentionally clearing something on their clipboard that they wanted to keep.

May be better to have an explicit way of copying [button or region? click here to copy etc], or perhaps give the ability to highlight and right click as well.

**Comment by hi-im-alex** [comment]
Good point I'll move this to users clicking on the hex or RGB only, and I'll also add a pop-up bubble to notify users they can click to copy.

I'll finally add an effect to let users know it's been copied.

- [x] Add Click to Copy Notification
- [x] Move click to copy to only function on the colour code
- [ ] Add animation upon copying
